src/utils/utils.py

```diff
@@ -1029,51 +1031,6 @@ def flatten_tcc_info_across_xcds(file, xcds, tcc_channel_per_xcd):
     return df
 
 
-def total_xcds(archname, compute_partition):
```

Comment:
We might want to keep total_xcds on top of the new get_mi300_num_xcds() for any other archs even they don't have the concept of "xcd". Think about future arch, people still probably want to call the unified interface from total_xcds